### ISSUE TITLE
Images are not override when on Scrape Mode "0 - Create new XML"

### ISSUE THREAD

**Issue by verybadsoldier** [Universal-Rom-Tools/Universal-XML-Scraper#86]
I noticed that when I scrape a system using scrape mode "0 - Create new XML" then the gamelist.xml is recreated from scratch. But the downloaded images are not recreated, right?
So it means when having profile MIX and games had incomplete MIX images then no new images get created even when the MIX images are complete now in the DB?

Maybe whe using that scrape mode then everything (including images) should be recreated? At the moment I think I would have to manually delete the images if I want them refreshed.

**Comment by Universal-Rom-Tools** [comment]
Yep, I'm thinking about that. Now we have 2 "updates" solution (NEW games, and only picture). 

I think I can change the "0 mode" ;) to rescrape all ;)
